src/components/RocketCard.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { useDispatch } from 'react-redux';
```

Comment:
- [ ]  I think it will be more professional to resolve the issue that this linter is pointer at, linters are there to ensure that we write clean and bug-free code. **Hint: You can define the prop-types like it was done in MissionCard.js to resolve the issue**. ℹ️